FEXCore/Source/Utils/LogManager.cpp

```diff
@@ -10,25 +10,22 @@ tags: glue|log-manager
 #include <FEXCore/fextl/fmt.h>
 #include <FEXCore/fextl/vector.h>
```

Comment:
This header is probably not needed anymore.

Reply:
Removed